README.md

```diff
@@ -42,15 +42,12 @@
 To install the latest version:
 ```
 curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash && export SUPPRESS_GCLOUD_CREDS_WARNING=true
-./terra
+sudo mv terra /usr/local/bin
```

Comment:
There's a line later on in the README that says "This will install the Terra CLI in the current directory. Afterwards, you may want to add it to your $PATH directly or move it to a place that is already on your $PATH (e.g. /usr/local/bin)". I'd prefer it that way so that the use of `sudo` is an understood decision by the user.

Reply:
I added comment `# Optional: Move to /usr/local/bin` and deleted that paragraph.

My motivation: There's a wall of text that people might read for their first installation, but they can skip for subsequent installations:

![image](https://user-images.githubusercontent.com/10929390/159806021-5b7fe1e0-6a2c-4217-abaa-1da67e1c31c0.png)

For the things that need to be done **every** installation, I put them in one section for convenience:

![image](https://user-images.githubusercontent.com/10929390/159806226-661c3c1b-69e6-41a4-b373-8e93e38aee89.png)

WDYT? (I assume everyone runs `sudo mv terra /usr/local/bin`, is that not the case?)

Reply:
Nice! I like having a "straight to the point" section for subsequent installs. 

I think installing binaries to a location that then get added to your `PATH` is more common than sudo moving to /usr/local/bin. This is what `gcloud` suggests, for example: https://cloud.google.com/sdk/docs/install